MatrixSDK/Crypto/Verification/Requests/MXKeyVerificationRequest.h

```diff
@@ -68,39 +68,28 @@ Accept an incoming key verification request.
                      failure:(void(^ _Nullable)(NSError *error))failure;
 
 /**
- The cancellation reason, if any.
+ Current state of the request
  */
-@property (nonatomic, nullable) MXTransactionCancelCode *reasonCancelCode;
-
-// Current state
 @property (nonatomic, readonly) MXKeyVerificationRequestState state;
 
-// Original data for this request
-@property (nonatomic, readonly) MXEvent *event;
+/**
+ The cancellation reason, if state is cancelled
+ */
+@property (nonatomic, readonly, nullable) MXTransactionCancelCode *reasonCancelCode;
```

Comment:
Here and in the other protocols I am either moving required methods from the generic protocol into default implementation (when not required by every implementation) or changing them to be readonly. No methods are added or removed